orttraining/orttraining/test/python/dhp_parallel/orttraining_test_parallel_train_simple_model.py

```diff
@@ -0,0 +1,121 @@
+# Add onnxruntime path as system path.
+# Otherwise, "import onnxruntime" may fail.
+import os
+import sys
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
Is this really necessary?

Reply:
Yes. It's also used in other distributed tests.

---
In reply to: [552812854](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r552812854) [](ancestors = 552812854)